OrbitBase/Logging.cpp

```diff
@@ -4,9 +4,18 @@
 
 #include <OrbitBase/Logging.h>
 
+absl::Mutex log_file_mutex;
 std::ofstream log_file;
 
 void InitLogFile(const std::string& path) {
   CHECK(!log_file.is_open());
```

Comment:
add MutexLock here as well.

Reply:
Yes, thanks for noticing: I should use mutex in all places when working with the log file